modules/ROOT/pages/clauses/create.adoc

```diff
@@ -205,10 +205,61 @@ Nodes created: 2 +
 Properties set: 4
 |===
 
+[role=label--new-5.26]
+[[dynamic-create]]
+== CREATE nodes and relationships using dynamic node labels and relationship types
```

Comment:
This is a really long title, is it possible to make it shorter?

`CREATE using dynamic node labels and relationship types` perhaps?